examples/response_stream/public/containers/app/pages/page_reducer_stream/index.tsx

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useState, FC } from 'react';
+
+import { Chart, Settings, Axis, BarSeries, Position, ScaleType } from '@elastic/charts';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiBadge,
+  EuiButton,
+  EuiCheckbox,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiProgress,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+
+import { useFetchStream } from '@kbn/aiops-utils';
+
+import { ApiReducerStream } from '../../../../../common/api';
+import {
+  initialState,
+  resetStream,
+  reducerStreamReducer,
+} from '../../../../../common/api/reducer_stream/reducer';
+
+import { Page } from '../../../../components/page';
+
+import { useDeps } from '../../../../hooks/use_deps';
+
+import { getStatusMessage } from './get_status_message';
+
+export const PageReducerStream: FC = () => {
+  const {
+    core: { http, notifications },
+  } = useDeps();
+
+  const basePath = http?.basePath.get() ?? '';
+
+  const [simulateErrors, setSimulateErrors] = useState(false);
+
+  const { dispatch, start, cancel, data, error, isCancelled, isRunning } = useFetchStream<
+    ApiReducerStream,
+    typeof basePath
+  >(
+    `${basePath}/internal/response_stream/reducer_stream`,
+    { simulateErrors },
+    { reducer: reducerStreamReducer, initialState }
+  );
+
+  const { progress, entities } = data;
+
+  const onClickHandler = async () => {
+    if (isRunning) {
+      cancel();
+    } else {
+      dispatch(resetStream());
+      start();
+    }
+  };
+
+  useEffect(() => {
+    if (error) {
+      notifications.toasts.addDanger(error);
+    }
+  }, [error, notifications.toasts]);
+
+  const buttonLabel = isRunning
+    ? i18n.translate('xpack.response_stream.stopbuttonText', {
+        defaultMessage: 'Stop development',
+      })
+    : i18n.translate('xpack.response_stream.startbuttonText', {
+        defaultMessage: 'Start development',
+      });
+
+  return (
+    <Page title={'Reducer stream'}>
+      <EuiText>
+        <p>
+          This demonstrates a single endpoint with streaming support that sends Redux inspired
```

Comment:
Should this block be internationationalized as well? Since the buttons and other items on this page are.

Reply:
The `i18n` code was in there because it was code I brought over from the ML/AIOps plugin. I removed now all `i18n` code from this example plugin, it looks like these won't need translations. Fixed in [39279ba](https://github.com/elastic/kibana/pull/132590/commits/39279ba7ef7bee6796df7f9f41bdca6a5125a5cd).